python/lsst/ts/ataos/config_schema.py

```diff
@@ -0,0 +1,227 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_ATAOS/blob/master/schema/ATAOS.yaml
+# title must end with one or more spaces followed by the schema version, which must begin with "v"
+title: ATAOS v3
+description: Schema for ATAOS configuration files
+type: object
+properties:
+  correction_frequency:
+    description: The frequency which corrections should be applied (Hz).
+    type: number
+    default: 1
+  m1:
+    description: List of polynomial coefficients for M1 correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  m1_lut_elevation_limits:
+    description: >-
+        Elevation limits for the validity of the M1 pressure LUT. Any values
+        outside this range you be assigned the value at this position.
+    type: array
+    items:
+        type: number
+        minItems: 2
+        maxItems: 2
+    default:
+        - 0.
+        - 90.
+  m2:
+    description: List of polynomial coefficients for M2 correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  m2_lut_elevation_limits:
+    description: >-
+        Elevation limits for the validity of the M2 pressure LUT. Any values
+        outside this range you be assigned the value at this position.
+    type: array
+    items:
+        type: number
+        minItems: 2
+        maxItems: 2
+    default:
+        - 0.
+        - 90.
+  hexapod_x:
+    description: List of polynomial coefficients for hexapod x-correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  hexapod_lut_elevation_limits:
+    description: >-
+        Elevation limits for the validity of the hexapod position LUT. Any
+        values outside this range you be assigned the value at this position.
+        This range apply equally for all axis.
+    type: array
+    items:
+        type: number
+        minItems: 2
+        maxItems: 2
+    default:
+        - 0.
+        - 90.
+  hexapod_y:
+    description: List of polynomial coefficients for hexapod y-correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  hexapod_z:
+    description: List of polynomial coefficients for hexapod z-correction equation (focus).
+    type: array
+    items:
+      type: number
+    default: [0.]
+  hexapod_u:
+    description: List of polynomial coefficients for hexapod u-correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  hexapod_v:
+    description: List of polynomial coefficients for hexapod v-correction equation.
+    type: array
+    items:
+      type: number
+    default: [0.]
+  hexapod_sensitivity_matrix:
+    description: >-
+        A matrix to map the cross terms dependencies between hexapod axis.
```

Comment:
is this written up in a technote somewhere we can reference?

Reply:
No.... Could probably add it to the package documentation.